### PR TITLE
#509 python script to apply all existing xsl to nrt or to a specific …

### PR DESCRIPTION
**Pull request by rosiereflo**
…file + update existing xsl so that they do not modify the input if the modification was already done

### CODE REVIEW

**Comment by adriguir** [comment]
Cool ;)

File: util/envDynawo.sh [hunk 1]
```diff
@@ -1723,6 +1729,7 @@ deploy_dynawo() {
   cp -r $DYNAWO_NRT_DIFF_DIR/*.py sbin/nrt/nrt_diff
   cp -r $DYNAWO_NRT_DIR/nrt.py sbin/nrt/.
   cp -r $DYNAWO_NRT_DIR/resources sbin/nrt/.
+  cp -r $current_dir/xsl sbin/.
```
Comment (gautierbureau) [inline util/envDynawo.sh]: could it be useful to add it in the distrib ?

**Comment by gautierbureau** [comment]
The README in util/xsl needs some update

File: util/envDynawo.sh [hunk 2]
```diff
@@ -1915,6 +1922,7 @@ create_distrib() {
   zip -r -y $ZIP_FILE bin/ lib/ share/ sources/
   zip -r -g -y $ZIP_FILE ddb/*.$DYNAWO_SHARED_LIBRARY_SUFFIX ddb/*.desc.xml ddb/*.extvar
   zip -r -g -y $ZIP_FILE sbin/curvesToHtml
+  zip -r -g -y $ZIP_FILE sbin/xsl
```
Comment (gautierbureau) [inline util/envDynawo.sh]: To add in create_modelica_distrib also
And the script need nry.py also ?

Comment (rosiereflo) [reply util/envDynawo.sh]: Not needed in create_modelica_distrib as we zip the whole sbin folder 
I am going to add the nrt script

**Comment by gautierbureau** [comment]
> The README in util/xsl needs some update

Did you do it ?

**Pull request merged**